WordPress/Classes/ViewRelated/Notifications/Views/NotificationDetailUserView.swift

```diff
@@ -0,0 +1,174 @@
+import SwiftUI
+import DesignSystem
+
+struct NotificationDetailUserView: View {
+	typealias FollowConfig = NotificationDetailUserView.FollowActionConfiguration
+
+    let config: Configuration
+
+    init(avatarURL: URL?, username: String?, blog: String?, onUserClicked: @escaping () -> Void) {
+        self.config = Configuration(avatarURL: avatarURL, username: username, blog: blog, onUserClicked: onUserClicked)
+    }
+
+    init(
+        avatarURL: URL?,
+        username: String?,
+        blog: String?,
+        isFollowed: Bool,
```

Comment:
Just an idea: If all these come from a single source, like `Comment` or `Blog` entity, I think we can have another init that accepts that type and assigns to the relevant fields.

Reply:
Did the same for `LikeUserTableViewCell` as well as for `NoteBlockUserTableViewCell` here https://github.com/wordpress-mobile/WordPress-iOS/pull/23178/commits/5d932346ed332cd4001d6999c08284b8711028d7